holmes_operator/scheduler/manager.py

```
@@ -0,0 +1,305 @@
+            job = self.scheduler.add_job(
+                func=execute_scheduled_check,
+                trigger=trigger,
+                args=(name, namespace, spec, scheduled_uid, self.k8s_api),
+                id=key,
+                replace_existing=True,
+                name=f"ScheduledHealthCheck {key}",
+            )
+
+            self.job_registry[key] = job.id
+            logger.info(f"Registered schedule for {key} with cron: {cron_expr}")
+            return job.id
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)